lib/utilities/string_extension.dart

```diff
@@ -0,0 +1,6 @@
+extension StringExtension on String {
+  String toFirstLetterUpperCase() {

```

Comment:
kindly read this [conversation](https://ffufmanilabootcamp.slack.com/archives/C027M886THS/p1679636361031749?thread_ts=1679635559.296549&cid=C027M886THS) 🙂

Reply:
resolved